scripts/generate-docs-examples.js

```diff
@@ -142,7 +140,15 @@ console.log(response${getResponsePostfix(i)})
   }
 
   code += '// END\n}'
-  const { results } = standard.lintTextSync(code, { fix: true })
+  const { results } = await new Promise((resolve, reject) => {
+    tsStandard.lintText(code, { fix: true }, (err, result) => {
+      if (err) {
+          reject(err)
+      } else {
+          resolve(result)
+      }
+    })
+  })
```

Comment:
I don't think we need to wrap this explicitly in a Promise. Since you made `generateAsciidoc` an async function, any errors thrown by `lintText` will be thrown just like any other exception would be.

```suggestion
  const { results } = await tsStandard.lintText(code, { fix: true })
```